### PR TITLE
wxRuby3 1.0.0 release post

### PR DESCRIPTION
**Pull request by mcorino**

### CODE REVIEW

File: _posts/2024-05-18-wxruby3-1.0.0-released.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "wxRuby3 1.0.0 released"
+date: 2024-05-18
+comments: true
+---
+
+It's time! wxRuby3 has reached it's 1.0.0 stable release.
```
Comment (PBfordev) [inline _posts/2024-05-18-wxruby3-1.0.0-released.md:7]:
```suggestion
It's time! wxRuby3 has reached its 1.0.0 stable release.
```

File: _posts/2024-05-18-wxruby3-1.0.0-released.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+This release targets the wxWidgets 3.2.5 release by default, which contains (among many others) quite some fixes which 
+came from the development of wxRuby3, and can be installed from [RubyGems](https://rubygems.org/gems/wxruby3) or 
+downloaded from [Github](https://github.com/mcorino/wxRuby3/releases/download/v1.0.0/wxruby3-1.0.0.gem). 
+
+This release is the result of about 1.5 years of development, starting at the abandoned remains of wxRuby2, taking a 
+page out of the wxPython Phoenix book and ending with this release.
+
+wxRuby3 is a cross-platform GUI library for Ruby, based on the mature wxWidgets GUI toolkit for C++. It uses native 
+widgets wherever possible, providing the correct look, feel and behavior to GUI applications on Windows, OS X and
```
Comment (PBfordev) [inline _posts/2024-05-18-wxruby3-1.0.0-released.md:17]:
```suggestion
widgets wherever possible, providing the correct look, feel and behavior to GUI applications on Windows, macOS and 
```

File: _posts/2024-05-18-wxruby3-1.0.0-released.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+downloaded from [Github](https://github.com/mcorino/wxRuby3/releases/download/v1.0.0/wxruby3-1.0.0.gem). 
+
+This release is the result of about 1.5 years of development, starting at the abandoned remains of wxRuby2, taking a 
+page out of the wxPython Phoenix book and ending with this release.
+
+wxRuby3 is a cross-platform GUI library for Ruby, based on the mature wxWidgets GUI toolkit for C++. It uses native 
+widgets wherever possible, providing the correct look, feel and behavior to GUI applications on Windows, OS X and 
+Linux/GTK. wxRuby aims to provide a comprehensive solution to developing professional-standard desktop applications in 
+Ruby.
+wxRuby3 also provides extensive usage documentation at it's [Wiki](https://github.com/mcorino/wxRuby3/wiki) as well as
```
Comment (PBfordev) [inline _posts/2024-05-18-wxruby3-1.0.0-released.md:20]:
```suggestion
wxRuby3 also provides extensive usage documentation at its [Wiki](https://github.com/mcorino/wxRuby3/wiki) as well as 
```

File: _posts/2024-05-18-wxruby3-1.0.0-released.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+widgets wherever possible, providing the correct look, feel and behavior to GUI applications on Windows, OS X and 
+Linux/GTK. wxRuby aims to provide a comprehensive solution to developing professional-standard desktop applications in 
+Ruby.
+wxRuby3 also provides extensive usage documentation at it's [Wiki](https://github.com/mcorino/wxRuby3/wiki) as well as 
+a complete [API reference](https://mcorino.github.io/wxRuby3/index.html). 
+
+## What has changed in this release?
+
+* improved documentation
+* support building against wxWidgets HEAD version
```
Comment (PBfordev) [inline _posts/2024-05-18-wxruby3-1.0.0-released.md:26]: ``
```suggestion
* support building against wxWidgets GIT master version
```

**Review by PBfordev — COMMENTED** [review]
Sorry, just few typos.

**Comment by vadz** [comment]
Thanks!

**Pull request closed without merging**